src/Compiler/FSComp.txt

```diff
@@ -1745,4 +1745,5 @@ featureReuseSameFieldsInStructUnions,"Share underlying fields in a [<Struct>] di
 3862,parsStaticMemberImcompleteSyntax,"Incomplete declaration of a static construct. Use 'static let','static do','static member' or 'static val' for declaration."
 3863,parsExpectingField,"Expecting record field"
 3864,tooManyMethodsInDotNetTypeWritingAssembly,"The type '%s' has too many methods. Found: '%d', maximum: '%d'"
-3865,parsOnlySimplePatternsAreAllowedInConstructors,"Only simple patterns are allowed in primary constructors"
\ No newline at end of file
+3865,parsOnlySimplePatternsAreAllowedInConstructors,"Only simple patterns are allowed in primary constructors"
+3866,chkStaticAbstractInterfaceMembers,"A static abstract interface member can be accessed only on a type parameter."
```

Comment:
I think this might sound a bit more specific:
```suggestion
3866,chkStaticAbstractInterfaceMembers,"A static abstract non-virtual interface member should only be called via type parameter."
```

Also, an extra mile can be adding a parameter to this message, `3866,chkStaticAbstractInterfaceMembers,"A static abstract non-virtual interface member should only be called via type parameter (for example: %s)."`, and pass a one-liner of which method user is trying to call, like "'T.Parse(...)".

Reply:
@vzarytovskii  Done. See my last commit.